### PR TITLE
Containerizing Redis Cluster

### PR DESCRIPTION
**Pull request by shah-zobair**
#### What does this PR do?

Containerizing Redis Cluster
#### How should this be manually tested?

See the readme.md
#### Is there a relevant Issue open for this?

It's a demo
#### Who would you like to review this?

/cc @etsauer @shraderdm

### CODE REVIEW

File: demos/redis-cluster/Implementation Guide [hunk 1]
```diff
@@ -0,0 +1,115 @@
+Creating image and pushing to openshift registry.
```
Comment (etsauer) [inline demos/redis-cluster/Implementation Guide]: This should be in markdown format and renamed Implementation_Guide.adoc. Then should be linked from somewhere in the README.

**Comment by etsauer** [comment]
@shah-zobair just one comment above, otherwise looks good.

**Comment by shah-zobair** [comment]
@etsauer : Renamed the "Implementation Guide". Actually, I have included everything in Readme and I think the Implementation Guide is not necessary anymore.

Is there any better way to parameterize the Registry URL inside the yaml files?

**Comment by etsauer** [comment]
The best way is to pull the image URL out of the deployment Configs and into a common ImageStream, then have all the yaml files reference that image stream. At least then it's hard coded in fewer places. Then the instructions to configure it for a particular cluster can be simpler.

**Pull request merged**